### DISCUSSION TITLE
Tandoor with separate DB and Nginx Proxy Manager

### DISCUSSION THREAD

**Discussion by plumped** [TandoorRecipes/recipes#2499] [Q&A]
Hi everyone

did somebody get the docker-compose to run without the built in nginx container?
Im using the nginx proxymanager for all my apps, and wanted to use it with tandoor too but after several hours and input from chat gpt im giving up. Hope somebody had better experience?

i just want tandoor as standalone container and connect to db over the env variables and reverse prox over npm.

any ideas?

**Comment by ARS-CS** [comment]
I'm using Nginx Proxy Manager too but, if I'm not mistaken, it is still recommended to use the built-in Nginx container. [Read this thread for more info](https://github.com/TandoorRecipes/recipes/discussions/1163).
